fusion/Modules/Lua/cryptomatte_utilities.lua

```diff
@@ -908,6 +908,19 @@ function module.create_matte_image(input_image, layer_images, manifest, matte_na
             matte_values[0.0] = true
         else
             local matte_id = manifest[matte_name]
+
+            if string.match(matte_name, "*") then
+                wildcard = string.gsub(matte_name, "*", ".*.?")
+                for crypto_name, __ in pairs(manifest) do
+                    wildcard_match = string.match(crypto_name, wildcard)
+                    if wildcard_match == crypto_name then
+                        matte_id = manifest[crypto_name]
+                        local matte_value = module._hex_to_float(matte_id)
+                        matte_values[matte_value] = true
+                    end
+                end
+            end
```

Comment:
Hi @jamesdeschenes 

Looks like a solid idea you got there!

My only comment would be to isolate this wildcard solving logic and call it before using this function. Primary reason is to split responsibilities and keep things simple. One function to solve wildcards to their matched absolute results, and this function can stay as is considering the `matte_names` argument.

WDYT?